programs/ssl/ssl_client2.c

```diff
@@ -389,6 +397,30 @@ int main( void )
 #define USAGE_SERIALIZATION ""
 #endif
 
+#if defined(MBEDTLS_ZERO_RTT)
```

Comment:
Same here: TLS 1.3 guard + info string indicating that this is TLS 1.3 specific?